src/aggregator/client/queue.go

```diff
@@ -128,8 +128,10 @@ type queue struct {
 
 func newInstanceQueue(instance placement.Instance, opts Options) instanceQueue {
 	var (
-		instrumentOpts     = opts.InstrumentOptions()
-		scope              = instrumentOpts.MetricsScope()
+		instrumentOpts = opts.InstrumentOptions()
+		scope          = instrumentOpts.MetricsScope().Tagged(map[string]string{
+			"endpoint": instance.Endpoint(),
```

Comment:
Do we want endpoint or `ID()` here? `ID()` seems more debuggable generally to me.

Reply:
this will be pretty bad in terms of cardinality, you only want conn metrics - not on all scope

Reply:
Updated: used tagged scope only for queue metrics.